docs/topics/auth/customizing.txt

```diff
@@ -400,10 +400,9 @@ the Django model that you wish to use as your user model.
 Using a custom user model when starting a project
 -------------------------------------------------
 
-If you're starting a new project, it's highly recommended to set up a custom
-user model, even if the default :class:`~django.contrib.auth.models.User` model
-is sufficient for you. This model behaves identically to the default user
-model, but you'll be able to customize it in the future if the need arises::
+If you're starting a new project, you can set up a custom user model, that
+behaves identically to the default user model by subclassing
+:class:`~django.contrib.auth.models.AbstractUser`::
```

Comment:
I like this change. But, considering the PR title, I feel entitled to suggest that we consider going one step further and take the initiative to not have the annoying `pass` class definition in the example. This means being more tutorial-ly and less reference-y (and I think it's fine since we are in "topics").
```suggestion
If you're starting a new project, you can set up a custom user model that
behaves identically to the default user model by subclassing
:class:`~django.contrib.auth.models.AbstractUser`. For example:

.. code-block:: python
    :caption: ``myapp/models.py``

    from django.contrib.auth.models import AbstractUser
    from django.db import models
    from django.db.models import Case, Value, When
    from django.db.models.functions import Concat


    class User(AbstractUser):
        display_name = models.GeneratedField(
            expression=Case(
                When(first_name="", then="last_name"),
                When(last_name="", then="first_name"),
                default=Concat("first_name", Value(" "), "last_name"),
            ),
            output_field=models.TextField(),
            db_persist=False,
        )
```

Reply:
This no longer _"behaves identically to the default user model"_. 

I wouldn't add this, personally. `display_name` here is just duplicating `get_full_name()`, and in so doing leveraging the very `first_name`, `last_name` fields that we want to get rid of.

Reply:
TBH I don't want to add anything here. 

I think custom user models should just fade away slowly as we work on auth.User. 

I think by showing more advanced examples we just encourage people into what I consider a mistake. 

I also think adding content here adjusts to scope of what I've said we should implement here, which has wide agreement as far as I can see, and into territory where opinions will begin to differ, and which I've not argued for, and for which I'm not sure we've agreed a route forward. 

I don't want to flex the scope like that. I'd rather keep it as small as possible. 

Does that make sense?

Reply:
Yes. I don't think the existing docs are that great. 

I just don't want to rewrite them at length here. 🫠

Reply:
Fair enough! Merging then.

Reply:
(Well I will first push the removal of the comma before "that" because I feel that reads weird as proposed...)

Reply:
Thanks @nessita 🙏